cfn-resources/test/e2e/utility/aws_helper.go

```diff
@@ -75,14 +76,15 @@ func waitForStackCreateComplete(svc *cfn.Client, stackID string) (*cfn.DescribeS
 		switch statusStr {
 		case "CREATE_COMPLETE":
 			return resp, nil
-		case "CREATE_FAILED":
+		case "CREATE_FAILED", "ROLLBACK_COMPLETE":
 			return nil, fmt.Errorf("stack status: %s : %s", statusStr, *resp.Stacks[0].StackStatusReason)
 		}
-		time.Sleep(3 * time.Second)
+		time.Sleep(2 * time.Second)
```

Comment:
not-blocking since `2 * time.Second` is in a loop every iteration of the loop creates a new instance, usually is better to define these outside the loops and with a clear name `const createWait = 2 * time.Second`

Reply:
added `stackStatusWait` const and re-used